model/patch/patch.go

```diff
@@ -974,32 +976,40 @@ func (p PatchesByCreateTime) Swap(i, j int) {
 	p[i], p[j] = p[j], p[i]
 }
 
-type PatchesByStatus []Patch
-
-func (p PatchesByStatus) Len() int {
-	return len(p)
-}
-
-func (p PatchesByStatus) Less(i, j int) bool {
-	return p[i].patchStatusPriority() < p[j].patchStatusPriority()
-}
+func GetCollectiveStatus(allPatches []Patch) string {
+	hasCreated := false
+	hasFailure := false
+	hasSuccess := false
+
+	for _, p := range allPatches {
+		switch p.Status {
+		case evergreen.PatchStarted:
+			return evergreen.PatchStarted
+		case evergreen.PatchCreated:
+			hasCreated = true
+		case evergreen.PatchFailed:
+			hasFailure = true
+		case evergreen.PatchSucceeded:
+			hasSuccess = true
+		}
+	}
 
-func (p PatchesByStatus) Swap(i, j int) {
-	p[i], p[j] = p[j], p[i]
-}
+	if !(hasCreated || hasFailure || hasSuccess) {
+		grip.Critical(message.WrapError(errors.New("unknown patch status"), message.Fields{
```

Comment:
I think we can drop the message.WrapError. We don't need the stack here because the cause is very distinct.

Reply:
true. done.